packages/ottehr-components/lib/components/CustomContainerFactory.tsx

```diff
@@ -124,26 +123,27 @@ export const CustomContainer: FC<ContainerProps> = ({
               mx: { xs: 'auto', md: 2 },
             }}
           >
-            <Box
-              sx={{
-                display: 'flex',
-                alignItems: 'center',
-                cursor: 'pointer',
-                mr: 1,
-                backgroundColor: 'rgba(226, 240, 255, 0.2)',
-                padding: '12px',
-                border: 'none',
-                borderRadius: '50%',
-                '&:hover': {
-                  backgroundColor: 'rgba(226, 240, 255, 0.5)',
-                },
-              }}
-              onClick={() => {
-                navigate('/patient-portal');
-              }}
-            >
-              <img src={portal} alt="portal" />
-            </Box>
+            <Tooltip title="Patient Portal" arrow>
+              <Link to="/patient-portal">
+                <Box
+                  sx={{
+                    display: 'flex',
+                    alignItems: 'center',
+                    cursor: 'pointer',
+                    mr: 1,
+                    backgroundColor: 'rgba(226, 240, 255, 0.2)',
+                    padding: '12px',
+                    border: 'none',
+                    borderRadius: '50%',
+                    '&:hover': {
+                      backgroundColor: 'rgba(226, 240, 255, 0.5)',
+                    },
+                  }}
+                >
+                  <img src={portal} alt="portal" />
```

Comment:
please improve the alt attribute

Reply:
Please read this to learn about what an alt tag is for https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt

Reply:
Please read this about alt tags Please read this to learn about what an alt tag is for https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt

Reply:
Please read this about alt tags https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/alt